lib/chrome.js

```diff
@@ -126,15 +123,28 @@ export class StorageArea {
         console.error(`Sync interval should be at least ${MIN_INTERVAL} milliseconds`);
         return;
       }
-      this._sync().then(() => {
-        this.syncTimer = setInterval(() => {
-          this._sync().catch(err => {
-            console.error("Sync Error", err.message);
-          });
-        }, config.interval);
-      }, err => {
-        console.error("Sync Disabled", err.message);
-      });
+
+      // Start sync
+      if (!this.syncTimer) {
+        if (config.type !== "kinto") {
+          throw new Error("Unsupported type, try setting chrome.storage.sync.config.type = \"kinto\".");
+        }
+        this._sync().then(() => {
+          this.syncTimer = setInterval(() => {
+            this._sync().catch(err => {
+              console.error("Sync Error", err.message);
+            });
+          }, config.interval);
+        }, err => {
+          console.error("Sync Disabled", err.message);
```

Comment:
Nit: I don't think this is a situation where sync is *disabled*, but rather one where a single sync operation has failed.